### ISSUE TITLE
Style error: council-of-science-editors-alphabetical.csl

### ISSUE THREAD

**Issue by JeremyHorst** [citation-style-language/styles#2749]
The journal's issue number should appear if it exists, and there should be a period after the journal name abbreviation, for CSE 8.
This is either a problem with this format file, or Papers3... help!

**Comment by damnation333** [comment]
Hi @JeremyHorst
Can you post an example article here or a copy of the guidelines or something? Just something we can base this on. Thanks.

**Comment by JeremyHorst** [comment]
It currently leads to the output:
Horst JA, Ellenikiotis H, Milgrom PL. 2016. UCSF Protocol for Caries Arrest
Using Silver Diamine Fluoride: Rationale, Indications and Consent. J Calif
Dent Assoc 44:16–28.

Which should be:
Horst JA, Ellenikiotis H, Milgrom PL. 2016. UCSF Protocol for Caries Arrest
Using Silver Diamine Fluoride: Rationale, Indications and Consent. J Calif
Dent Assoc. 44(1):16–28.

Subtle but important difference... would lead to extensive unnecessary
manual curation.




Peace and Thanks,
Jeremy

---
*Jeremy Horst DDS, PhD*
Postdoctoral fellow, UCSF DeRisi Lab
<https://scholar.google.com/citations?user=CetD6iHud9IC&hl=en>
<https://www.linkedin.com/in/jeremyhorst>
<https://www.youtube.com/playlist?list=PLwbU_gwsxa0FmnREk9AYHlE7_kDL-I1dg>

<http://profiles.ucsf.edu/jeremy.horst>

**Comment by adam3smith** [comment]
Thanks! The question was for the relevant passage from the CSE manual, though, so we can double-check. We're always making sure to go back to style guides before making changes to styles just to be safe.

**Comment by rmzelle** [comment]
> It currently leads to the output: Horst JA, Ellenikiotis H, Milgrom PL. 2016. UCSF Protocol for Caries Arrest Using Silver Diamine Fluoride: Rationale, Indications and Consent. J Calif Dent Assoc 44:16–28.

I'm a bit confused here, because https://github.com/citation-style-language/styles/blob/master/council-of-science-editors-alphabetical.csl currently doesn't seem to print the journal title for articles (see @damnation333's example above).

**Comment by adam3smith** [comment]
(note this is with papers 3 -- might just be an old version of the style?)

**Comment by rmzelle** [comment]
@adam3smith, that might be the case, but that doesn't take away the fact that our current style doesn't seem to print the journal title for articles, which can't be right.

**Comment by adam3smith** [comment]
yes, of course. Two separate issues. We need to fix the style, but the error isn't what was originally reported.

**Comment by rmzelle** [comment]
Right.

**Comment by POBrien333** [comment]
We've recently updated the CSE author-date style, but the numeric style is still from 2014.
Could I just copy the bibliography part over from the author-date style?
Are the two styles in alignment and just the in-text bit is different? I can only assume this style gets used widely and it would be good to get it fixed asap.

**Comment by rmzelle** [comment]
> Could I just copy the bibliography part over from the author-date style?
> Are the two styles in alignment and just the in-text bit is different?

We really need to check this with the [manual](https://www.councilscienceeditors.org/publications/scientific-style-and-format/), or with somebody who has access to it, especially since the 8th edition apparently changed the citation format quite a bit.

**Comment by POBrien333** [comment]
I have uploaded the pdf to the manual in the other style when I fixed it recently ;)

**Comment by rmzelle** [comment]
@POBrien333, can you remove that link? We really shouldn't distribute commercial style guide materials ourselves.

**Comment by rmzelle** [comment]
> Are the two styles in alignment and just the in-text bit is different?

Since you have access to the manual, I assume this question is regarding the CSL styles, instead of the different CSE citation formats? If CSL styles are very similar, you can use something like https://www.diffchecker.com/ to compare two versions.

**Comment by adam3smith** [comment]
@POBrien333  has the manual, I believe (so do I)

**Comment by POBrien333** [comment]
Can we close this one?

**Comment by rmzelle** [comment]
Once https://github.com/citation-style-language/styles/issues/2749#issuecomment-312643457 has been addressed, yes.

**Comment by cazzerson** [comment]
The issue number is also missing in the output of the current council-of-science-editors-author-date.csl.

https://github.com/citation-style-language/styles/blob/95a681e06466dde8ffaa775c4e1f1f0209464a93/council-of-science-editors-author-date.csl#L248-L249

https://github.com/citation-style-language/styles/blob/master/council-of-science-editors-alphabetical.csl#L239

**Comment by adam3smith** [comment]
The missing
